panda-whale-vekm/dinehard
# Vicky/database

Description
- Cuisine selected by user gets stored into the cuisine table db.

Trello
- Task in progress

Testing Instructions
- Run NPM install to ensure system is up to date

Checklist
N/A

Additional Notes, Images, etc.
